packages/ui/src/icons/index.ts

```diff
@@ -7,6 +7,7 @@ export * from './checkmark-circle-icon';
 export * from './checkmark-icon';
 export * from './chevron-down-icon';
 export * from './chevron-up-icon';
+export * from './chevron-right-icon.web';
```

Comment:
I just added this export so that it will get picked up properly in `web.ts` and will be available in the extension. 

In `web.ts` we load all the icons from index with:
```
export * from './src/icons';
```

FYI I didn't name all the icons as `.web` yet as I was thinking we could possibly share them with mobile. It doesn't make much difference calling them `.web` or not, just explaining the rationale for having a few `.web` and most not